SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -1215,7 +1217,7 @@ void ActorSystem::ResolveObjectReferences(FRepLayout& RepLayout, UObject* Replic
 			{
```

Comment:
The fact we're using `PreNetReceive` instead of `PreReceiveSpatialUpdate` on in actorsystem::1167 feels like a bug waiting to happen, do we want to swap it out?

Reply:
Hm, interesting spot.
I see that the implementation of PreReceiveSpatialUpdate is mostly to ensure things will not crash and then it calls `PreNetReceive`, so I guess there could be some justification to call it directly, but yeah, I think I would prefer a `PreReceiveSpatialUpdate`.

Hmm, can't decide, feels like the function could be doing some unnecessary stuff with setting up the replicator, but at the same time, we should try to preserve the `prenetreceive`-`postnetreceive` pairing, and the `prereceivespatial`-`postreceivespatial` pairing.

Reply:
hmm, maybe @improbable-dmitrii has some thoughts